### PR TITLE
Added numeric buttons for calculator and made some UI change

### PR DESCRIPTION
**Pull request by nani67**

### CODE REVIEW

**Comment by nani67** [comment]
This commit adds Floating Action Buttons for the numeric keys and also makes some UI changes so that it looks like a bit of "Materialized iOS Calculator (kinda)"

**Comment by Bhavnaharitsa** [comment]
@nani67 Can you please add the screenshot?

**Comment by nani67** [comment]
I'm outside now. Will do it by 10pm

**Comment by Bhavnaharitsa** [comment]
@nani67 I think you can refer to the guidelines and rules of Open source. Please avoid unnecessary messages and need not create another pull request again just for screen shot of the changes you've done in this pr. Close the pr and paste it here just away.

**Comment by nani67** [comment]
This link would navigate to the screenshot of the UI change which I informed you earlier. 
(Sorry for that. I've never been with any open source project contribution so I dunno that)

[https://drive.google.com/file/d/1--baPofTALzn5aNKK-Mr-4kOCaNsBfqD/view?usp=drivesdk](url)

**Comment by Bhavnaharitsa** [comment]
@nani67 GitHub error 404

**Comment by Bhavnaharitsa** [comment]
Just waste it...dont take the burden to make it into links ..some times they work not right

File: pubspec.lock [hunk 1]
```diff
@@ -29,13 +43,27 @@ packages:
...
+    source: hosted
+    version: "2.1.3"
   cupertino_icons:
     dependency: "direct main"
     description:
       name: cupertino_icons
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.1.2"
+    version: "0.1.3"
```
Comment (Bhavnaharitsa) [inline pubspec.lock]: Lots of unnecchanges...you could make a new issue and pr for version updates

File: pubspec.lock [hunk 2]
```diff
@@ -46,41 +74,55 @@ packages:
...
-    version: "0.12.5"
+    version: "0.12.6"
   meta:
     dependency: transitive
     description:
       name: meta
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.1.6"
+    version: "1.1.8"
```
Comment (Bhavnaharitsa) [inline pubspec.lock]: Same

File: pubspec.lock [hunk 3]
```diff
@@ -46,41 +74,55 @@ packages:
...
+    version: "2.1.4"
   matcher:
     dependency: transitive
     description:
       name: matcher
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.12.5"
+    version: "0.12.6"
   meta:
```
Comment (Bhavnaharitsa) [inline pubspec.lock]: Same

**Comment by nani67** [comment]
Here is the screenshot of the app

https://drive.google.com/file/d/1--baPofTALzn5aNKK-Mr-4kOCaNsBfqD/view?usp=drivesdk

**Comment by nani67** [comment]
I'm closing this PR and instead, I'll create two of them (one for updated version and one for UI changes). This is because this particular PR has both of them.

**Pull request closed without merging**